examples/README.md

```diff
@@ -106,30 +106,30 @@ When you open a notebook, select the kernel `nvflare_example` using the dropdown
-| [Federated Statistics for Medical Imaging](./advanced/federated-statistics/image_stats/README.md)                                      | -            | [Image Histograms](./advanced/federated-statistics/image_stats.ipynb)                                                                                                  | see requirements.txt   | Example of gathering local image histogram to compute the global dataset histograms.                                                                            |
-| [Federated Statistics for DataFrame](./advanced/federated-statistics/df_stats/README.md)                                               | -            | [Data Frame Federated Statistics](./advanced/federated-statistics/df_stats.ipynb), [Visualization](./advanced/federated-statistics/df_stats/demo/visualization.ipynb)  | see requirements.txt   | Example of gathering local statistics summary from Pandas DataFrame to compute the global dataset statistics.                                                   |
+| Example                                                                                                                                |  Notebooks                                                                                                                                                             | Notes                  | Summary                                                                                                                                                         |
+|----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Federated Statistics Overview](./advanced/federated-statistics/README.md)                                                             | -                                                                                                                                                                      | -                      | Discuss the overall federated statistics features.                                                                                                              |
```

Comment:
Should we have this entry inside a table?
I am thinking maybe we just don't have this row?

Reply:
Some people had strong opinions we should have things formatted in a table, so we converted everything to table. If we have a good way to use columns to categorize and distinguish how the different entries are different, that would be great, but for some things it was not obvious to me the best way to do that.